plugin/storage/kafka/options.go

```diff
@@ -43,13 +43,17 @@ const (
 	suffixCompression      = ".compression"
 	suffixCompressionLevel = ".compression-level"
 	suffixProtocolVersion  = ".protocol-version"
+	suffixLingerMS         = ".linger-ms"
+	suffixBatchSize        = ".batch-size"
 
 	defaultBroker           = "127.0.0.1:9092"
 	defaultTopic            = "jaeger-spans"
 	defaultEncoding         = EncodingProto
 	defaultRequiredAcks     = "local"
 	defaultCompression      = "none"
 	defaultCompressionLevel = 0
+	defaultLingerMS         = 10
+	defaultBatchSize        = 1024
```

Comment:
I think it would be better to be consistent with the kafka defaults to avoid confusion, and then document values that produce better performance? So `linger.ms` default to `0` and `batch.size` to `16384`.

Reply:
Yes, I agree.

Reply:
Changed to the suggested values in the latest commit.